contracts/examples/PullPaymentCapableExample.sol

```diff
@@ -0,0 +1,9 @@
+import '../PullPaymentCapable.sol';
+
+// Example class using PullPaymentCapable
+contract PullPaymentCapableExample is PullPaymentCapable {
```

Comment:
Do you prefer for tests to have this kind of structure? Where instead of testing the contract itself, an example contract that implements it is created.